lib/utils.js

```diff
@@ -30,7 +30,7 @@ function Utils(config) {
                 var feature;
                 if (val) {
                     try {
-                        feature = geobuf.geobufToFeature(val);
+                        feature = geobuf.decode(new Pbf(val));
```

Comment:
We should make sure we have a plan for what the migrations and uptime are going to look like before we mix the geobuf upgrade into this.